manifests/postgres/sql/create-run-databases.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+psql -v ON_ERROR_STOP=1 --username "$POSTGRES_USER" --dbname "$POSTGRES_DB" <<-EOSQL
```

Comment:
Can you put the schema into a separate file that is pulled into this script?  The maintained helm charts are able to read in a raw sql file to generate the initial schema and I'd like to use that logic long term.

Reply:
> raw sql file
Unfortunately, this isn't a raw sql script, for example, `\c` is a way to change the database, but we haven't figured out how to do this in raw sql.

Reply:
This commit 0a6411ab5a241072df8633a660540f5dcf424bee solves this issue